docs/source/traces.rst

```diff
@@ -146,6 +146,24 @@ Get
 
     $ st2 trace get <trace-id>
 
+* View the causation chain in a trace for an action execution. Similarly for rule and trigger-instance.
```

Comment:
Would also be good to explain what causation (and causation) chain is in this context :trollface: